### ISSUE TITLE
Define a GH Action for automated release publishing

### ISSUE THREAD

**Issue by sco1** [okken/pytest-check#31]
I noticed [a comment](https://github.com/okken/pytest-check/pull/26#issuecomment-567166186) in #26 that you'd like to configure this project to automate publishing to PyPI.

If you're interested, there is a [GitHub Action under the PyPA umbrella](https://github.com/pypa/gh-action-pypi-publish) domain that will publish distribution files to PyPA when the action is triggered.

You can find PyPA's guide [here](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/), and here's an example of the workflow I use for [`flake8-annotations`](https://github.com/python-discord/flake8-annotations/tree/master/.github/workflows) to build & publish a release to PyPI every time a new release is published on GitHub:

```yml
name: Publish to PyPI

on:
  release:
    types: [published]

jobs:
  build:
    name: Build dist & publish
    runs-on: ubuntu-18.04
    
    steps:
    - uses: actions/checkout@v1
    - name: Set up Python
      uses: actions/setup-python@v1
      with:
        python-version: '3.x'

    - name: Install build dependencies & build
      run: |
        python -m pip install --upgrade pip
        pip install setuptools wheel
        python setup.py sdist bdist_wheel
      
    - name: Publish package
      uses: pypa/gh-action-pypi-publish@v1.0.0a0
      with:
        user: __token__
        password: ${{ secrets.pypi_api_token }}
```

**Comment by okken** [comment]
thanks for the info

**Comment by okken** [comment]
this isn't really an issue and I don't want to track it in issues, so closing for now.
